src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.tsx

```diff
@@ -54,13 +55,16 @@ export const getDaysOpen = (incident: IncidentListItem) => {
   return differenceInCalendarDays(new Date(), createdAtDate)
 }
 
-const Td: FunctionComponent<{ detailLink: string }> = ({
+const Td: FunctionComponent<{ detailLink: string; isFocusable?: boolean }> = ({
   detailLink,
   children,
+  isFocusable = false,
   ...rest
 }) => (
   <TdStyle {...rest}>
-    <Link to={detailLink}>{children}</Link>
+    <Link to={detailLink} tabIndex={isFocusable ? 0 : -1}>
```

Comment:
Each cell has a link that points to the same location. That requires a lot of tabbing.

Reply:
Because only a single `<td />` in a row gets the `isFocusable` prop, there's only a single tabbable link per row.